client/src/main/java/org/opensearch/ml/client/MachineLearningNodeClient.java

```diff
@@ -228,6 +232,14 @@ public void deploy(String modelId, ActionListener<MLDeployModelResponse> listene
         }));
     }
 
+    @Override
+    public void createConnector(MLCreateConnectorInput mlCreateConnectorInput, ActionListener<MLCreateConnectorResponse> listener) {
+        MLCreateConnectorRequest createConnectorRequest = new MLCreateConnectorRequest(mlCreateConnectorInput);
+        client.execute(MLCreateConnectorAction.INSTANCE, createConnectorRequest, ActionListener.wrap(listener::onResponse, e -> {
+            listener.onFailure(e);
+        }));
```

Comment:
Can we try to address this check warning?

Reply:
Didn't get you here. Can you elaborate more?

Reply:
![Screenshot 2023-10-09 at 3 17 45 PM](https://github.com/opensearch-project/ml-commons/assets/109556906/4ada69ba-d48e-4f7e-a28f-483d6799bd7d)

Can we cover the missing line?

Reply:
I change it to just `[listener](https://github.com/opensearch-project/ml-commons/pull/1437/files#diff-107fa188b5bb13651eed59ff14d5e1cdf8feb8b1e4091f32c7ce88969f68bfd3R238)`  based on this [comment](https://github.com/opensearch-project/ml-commons/pull/1437#discussion_r1347900118).
Also, I don't see this check on my file diff. Is there anything I have to do to enable it?

Reply:
you can see from [here](https://app.codecov.io/gh/opensearch-project/ml-commons/pull/1437/blob/client/src/main/java/org/opensearch/ml/client/MachineLearningNodeClient.java)

Reply:
Looks green to me [here](https://app.codecov.io/gh/opensearch-project/ml-commons/pull/1437/blob/client/src/main/java/org/opensearch/ml/client/MachineLearningNodeClient.java#L238)